### ISSUE TITLE
Can't log into EventLog message with custom event id

### ISSUE THREAD

**Issue by explodec14** [coreybutler/node-windows#95]
hey-
I am trying to store in EventViewer a log message with custom event id (different from 1000). i don't see my log get updated with this message.

Anyone has any clue why?

**Comment by coreybutler** [comment]
Is your process running with elevated administrative privileges? This is a requirement on Windows.

**Comment by explodec14** [comment]
Yes. i opened power-shell as an administrator and ran the script.
tried to do: <code>log.info("hello world!", 1005); </code> but the message is not there in the Event viewer

**Comment by jdor12321** [comment]
Sorry to drag up an old issue but i am having the same issues with this. It will not log with a custom event id number but will if its left at its default.

**Comment by coreybutler** [comment]
I've found this to be different across different versions of Windows. Which version are you running, @jdor12321?

**Comment by jdor12321** [comment]
Hi @coreybutler. I'm running this on Windows 7 Enterprise with the hope of running it on Windows Server 2003 R2 or the next version up ( I can't recall the version of the server). I haven't tested this on the server yet as was hoping go get it on PC first before deploying.

**Comment by ianbeks** [comment]
Hi, I have the same problem. 1000 logs fine but any other ID does not. Yes I'm elevated. Windows 10.
